broadinstitute/orsp-pub
# Cric 1260 api

## Addresses
https://broadinstitute.atlassian.net/browse/CRIC-1260

## Changes
ORSP Portal Update - Implement new status/button for Submit to IRB

## Testing
1. Add a "Submitted to IRB" button to the left of "Reject"
2. Change the default "Pending" status on the right (in yellow) to "Pending ORSP Review" when someone submits a sample/data cohort
3. Add a new status (in orange/different color), "Pending IRB Review," when ORSP hits the new "Submitted to IRB" button on the left.

---
